cypress/e2e/base.spec.ts

```diff
@@ -20,4 +20,27 @@ describe('Visit directory page and verify users exists', () => {
   });
 });
 
+describe('Onboarding Page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/onboarding');
+  });
+
+  it('displays the correct heading', () => {
+    cy.contains('h1', 'Onboarding');
+  });
+
+  it('fills out the form successfully', () => {
+    cy.get('form').within(() => {
+      cy.get('textarea[name="bio"]').type('Lorem ipsum dolor sit amet.');
+      cy.get('input[name="twitter_username"]').type('test_twitter');
+      cy.get('input[name="linkedin_url"]').type('https://www.linkedin.com/in/test_linkedin');
+      cy.get('input[name="github_username"]').type('test_github');
+      cy.get('input[name="website"]').type('https://www.testwebsite.com');
+      cy.visit('http://localhost:3000/directory');
+    });
+
+    cy.url().should('equal', 'http://localhost:3000/directory');
+  });
+});
+
```

Comment:
I think you need to login first within the test to run this test correctly or intercept the request to add login data, I'm getting an error on the Cypress test since it visits `http://localhost:3000/onboarding` right away 

![image](https://user-images.githubusercontent.com/68509315/234387146-094bcd81-0ac6-4817-8542-05a22bd901cf.png)